### PR TITLE
#1867 fix markdown violation

### PR DESCRIPTION
**Pull request by nishaaaaaant**
I have fixed 2 `.md` files just for reference. 
If this PR looks good to you. I will do the same for all other documents.

### CODE REVIEW

File: contrib_templates/gemstones_template.md [hunk 1]
```diff
@@ -56,10 +59,12 @@ single asteriks: *may*
 
 double asteriks: **may**
```
Comment (SergeCroise) [inline contrib_templates/gemstones_template.md:60]:
```suggestion
double asterisk: **may**
```

File: contrib_templates/general_notes.md [hunk 2]
```diff
@@ -9,36 +9,43 @@ tags:
...
+## Prerequsites
+
+## Special Terminology
 
-## Special Terminology:
+## Process
 
-## Process:
   Detailed Process steps
   Try a link to a footnote/reference secton here
```
Comment (SergeCroise) [inline contrib_templates/general_notes.md:29]:
```suggestion
  Try a link to a footnote/reference section here
```

File: contrib_templates/general_notes.md [hunk 3]
```diff
@@ -9,36 +9,43 @@ tags:
...
 ---
-## Glossary:
 
-## REFERENCES:
+## Glossary
+
+## REFERENCES
+
 1. Footnotes/Reference links (URLs, manpage websites, etc)
 2. Links to scripts, code and custom programs (for posterity in some cases if the originals might disappear )
```
Comment (SergeCroise) [inline contrib_templates/general_notes.md:50]:
```suggestion
2. Links to scripts, code and custom programs (for posterity in some cases if the originals might disappear)
```

File: contrib_templates/gemstones_template.md [hunk 4]
```diff
@@ -56,10 +59,12 @@ single asteriks: *may*
...
+## Conclusions
+
     It works great!
 
 ---
 
-### References/Footnotes:
+### References/Footnotes
+
 [14](https://rockylinux.org) (or use a bulleted list, numbered list, etc)
```
Comment (SergeCroise) [inline contrib_templates/gemstones_template.md:70]:
```suggestion
[14](https://rockylinux.org) (or use a bulleted list, numbered list, etc.)
```

File: contrib_templates/gemstones_template.md [hunk 5]
```diff
@@ -10,10 +10,11 @@ tags:
...
 # First level header Introduction
+
 Description Here
 
- - https://docs.rockylinux.org/guides/contribute/
- - More detailed info: [RockyLinux First-Time Contributor Guide](https://docs.rockylinux.org/guides/contribute/beginners/#first-time-contributor-guide)
+- <https://docs.rockylinux.org/guides/contribute/>
+- More detailed info: [RockyLinux First-Time Contributor Guide](https://docs.rockylinux.org/guides/contribute/beginners/#first-time-contributor-guide)
 
 !!! warning
```
Comment (SergeCroise) [inline contrib_templates/gemstones_template.md:19]:
```suggestion
!!! warning "Warning"
```

**Review by SergeCroise — COMMENTED** [review]
- https://en.wiktionary.org/wiki/asterisk
- https://www.merriam-webster.com/dictionary/etc.
- parenthesis
- section
- admonition (easier to translate)

**Review by alemorvan — APPROVED** [review]
It's good to fix markdown linter.

**Comment by sspencerwire** [comment]
@nishaaaaaant When making these changes to other documents, make sure that the "violation" is actually a violation for the `material-mkdocs` version of markdown that we use. Some HTML elements, for instance are still allowed (links within the page to another portion of the page, etc.) Also, I'd recommend making these changes slowly as in don't do ALL of the documents in a single PR, but do a PR with a few in it and then let us review those first, comment and merge before doing more. I think that will help overall. Thank you!!

**Comment by nishaaaaaant** [comment]
Sure I understood the flow. Will only raise PR one by one. I’ll definitely
again take a look at materials-mkdocs version that is used in here.

**Pull request merged**